include/plat/arm/css/common/css_def.h

```diff
@@ -101,6 +101,14 @@
 						NSRAM_SIZE,	\
 						MT_DEVICE | MT_RW | MT_NS)
 
+#if defined(IMAGE_BL2U)
+#define CSS_MAP_SCP_BL2U		MAP_REGION_FLAT(		\
+						SCP_BL2U_BASE,		\
+						SCP_BL2U_LIMIT		\
+							- SCP_BL2U_BASE,\
+						MT_CODE | MT_SECURE)
```

Comment:
This region is used to load and authenticate SCP_BL2U. Hence the attributes of this region should be RW rather Executable.

Reply:
I have changed this and checked that firmware update still works as expected which it does